ruby/bob/bob.rb

```diff
@@ -0,0 +1,50 @@
+class Bob
+
+  def hey(remark)
+    if blank?(remark)
+      "Fine. Be that way!"
+    elsif other?(remark)
+      "Whatever."
+    elsif pure_question?(remark)
+      "Sure."
+    elsif yelling?(remark)
+      "Whoa, chill out!"
+    else
+      "Whatever."
```

Comment:
This is returning the same value as line 7. Instead of having two returns, leave it as the default value.